src/components/portrait/portrait.test.js

```diff
@@ -0,0 +1,281 @@
+];
+
+const PortraitComponent = ({ ...props }) => {
+  return <Portrait {...props} />;
+};
+
+const TooltipPortraitComponent = ({ ...props }) => {
+  return (
+    <Portrait
+      tooltipMessage="Rebecca Smith"
+      tooltipIsVisible
+      src="https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light"
+      {...props}
+    />
+  );
+};
+
+context("Tests for Portrait component", () => {
+  describe("check props for Portrait component", () => {
+    it.each([
+      [
+        "chris.barber@sage.com",
+        "https://www.gravatar.com/avatar/cceff1ad774bf89b1f75cb6e5005333c?s=40&d=404",
+      ],
+    ])("should check gravatar for Portrait components", (emailID, srcImage) => {
+      CypressMountWithProviders(<PortraitComponent gravatar={emailID} />);
+      portraitPreview()
+        .should("have.css", "height", "40px")
+        .and("have.css", "width", "40px");
+      portraitPreview().children().should("have.attr", "src", srcImage);
+    });
+
+    it.each([
+      "https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light",
+
+      "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404",
+    ])("should check custom image url for Portrait component ", (srcImage) => {
+      CypressMountWithProviders(<PortraitComponent src={srcImage} />);
+      portraitPreview()
+        .should("have.css", "height", "40px")
+        .and("have.css", "width", "40px");
+      portraitPreview().children().should("have.attr", "src", srcImage);
+    });
+
+    it.each([
+      ["XS", 24],
+      ["S", 32],
+      ["M", 40],
+      ["ML", 56],
+      ["L", 72],
+      ["XL", 104],
+      ["XXL", 128],
+    ])(
+      "should check %s size for Portrait component ",
+      (size, heightAndWidth) => {
+        CypressMountWithProviders(<PortraitComponent size={size} />);
+        portraitPreview()
+          .should("have.css", "height", `${heightAndWidth}px`)
+          .and("have.css", "width", `${heightAndWidth}px`);
+      }
+    );
+
+    it.each(testImage)(
+      "should check alt text for %s in Portrait component ",
+      (src) => {
+        CypressMountWithProviders(
+          <PortraitComponent alt="cypress-test" src={src} />
+        );
+        portraitAlt().should("have.attr", "alt", "cypress-test");
+      }
+    );
+
+    it.each([
+      ["square", "0px"],
+      ["circle", "50%"],
+    ])("should check %s shape for Portrait component ", (shape, radius) => {
+      CypressMountWithProviders(<PortraitComponent shape={shape} />);
+      portraitPreview().children().should("have.css", "border-radius", radius);
+    });
+
+    it.each([
```

Comment:
Similar to line 48, the syntax is not correct in this array. Each instance needs to be bounded by square brackets, e.g. `it.each([["filter"],["error"],...["alert"],])`